viziphant/unitary_event_analysis.py

```diff
@@ -7,14 +7,13 @@
 import matplotlib.pyplot as plt
 import string
 import elephant.unitary_event_analysis as ue
+from collections import namedtuple
+
+Figure = namedtuple("Figure", "ax0 ax1 ax2 ax3 ax4 ax5")
```

Comment:
better to rename it to UE specific name like FigureUE or FigureUnitaryEvent to avoid namespace clash with other modules in the future

Reply:
In fact, I wonder if it would be smart to give telling names to the axes. E.g., axis_rasterplot, axis_significance, axis_rates, axis_coincidences, axis_unitary_events

Reply:
right
then we can name it AxesUE and remove "axis_" prefix